### PR TITLE
Refactor registry-console template and vars

### PR DESCRIPTION
**Pull request by michaelgugino**
This commit refactors registry-console role to
create it's own template, updates the template to
utilize proper image name string, and conforms
component to utilizing oreg_url.

This commit also removes redundant openshift_hosted_templates
role as it is no longer needed with this commit.

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1608476
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1613647

### CODE REVIEW

File: roles/cockpit-ui/tasks/main.yml [hunk 1]
```diff
@@ -1,63 +1,70 @@
...
+  copy:
+    src: "registry-console.yaml"
+    dest: "{{ mktemp.stdout }}/registry-console.yaml"
 
-  # XXX: Required for items still using command
-  - name: Create temp directory for kubeconfig
-    command: mktemp -d /tmp/openshift-ansible-XXXXXX
-    register: mktemp
-    changed_when: False
+- name: Create or registry-console template
```
Comment (vrutkovs) [inline roles/cockpit-ui/tasks/main.yml]: typo?

File: roles/cockpit-ui/tasks/main.yml [hunk 2]
```diff
@@ -1,63 +1,70 @@
 ---
-- block:
+- name: Create local temp dir for registry-console template
+  local_action: command mktemp -d /tmp/openshift-ansible-XXXXXXX
```
Comment (vrutkovs) [inline roles/cockpit-ui/tasks/main.yml]: This should be a proper `command`, as the template is being copied to master

Comment (michaelgugino) [reply roles/cockpit-ui/tasks/main.yml]: Good catch.

**Comment by mtnbikenc** [comment]
@sdodson Is there any issue in making our own copy of this template?  This PR moves the registry-console template out of the files folder with the rest of the 3.11 origin templates.  When the templates are refreshed, it would put the file back, but it will drift from what we are actually using.  This brings up another question in that if these templates are of no value to openshift-ansible, do we need to continue to maintain them?

**Comment by sdodson** [comment]
This passed CI, but there's no way it's actually working, the image name still contains '${component}'
https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/9447/test_pull_request_openshift_ansible_extended_conformance_install/10484/build-log.txt

We just have nothing that checks for successful deployment.

```
TASK [cockpit-ui : Deploy registry-console] ************************************
task path: /usr/share/ansible/openshift-ansible/roles/cockpit-ui/tasks/main.yml:50
changed: [localhost] => {
    "changed": true, 
    "cmd": [
        "oc", 
        "new-app", 
        "--template=registry-console", 
        "-p", 
        "IMAGE_NAME=openshift/origin-${component}:48a1596", 
```

**Pull request merged**